@xen-orchestra/web-core/lib/components/table/ColumnTitle.vue

```diff
@@ -80,7 +80,7 @@ const currentInteraction = computed(() =>
   interactions.value.find(interaction => router.currentRoute.value.query[columnName] === interaction.id)
 )
 
-const columnName = `${tableName}__${props.id}`
+const columnName = stringifyColumnName(tableName, props.id)
```

Comment:
To keep _prop_ reactivity, this should be a _computed_:
```suggestion
const columnName = computed(() => stringifyColumnName(tableName, props.id))
```